### PR TITLE
Risk model docstrings

### PR DESCRIPTION
**Pull request by therealsilence**
- Update docstrings for risk:* model to clarify use.
- Update links in various intro docs.

### CODE REVIEW

File: synapse/models/risk.py [hunk 1]
```diff
@@ -445,13 +445,13 @@ def getModelDefs(self):
                     }),
                     ('type', ('risk:compromisetype', {}), {
                         'ex': 'cno.breach',
-                        'doc': 'The compromise type.',
+                        'doc': 'A user-specified type for the compromise',
```
Comment (Cisphyx) [inline synapse/models/risk.py:448]: missing period

File: synapse/models/risk.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ def getModelDefs(self):
                     'doc': 'An instance of a vulnerability present in a target.',
                 }),
                 ('risk:threat', ('guid', {}), {
-                    'doc': 'A threat cluster or subgraph of threat activity.',
+                    'doc': 'A threat cluster or subgraph of threat activity, as described by a specific reporter.',
```
Comment (invisig0th) [inline synapse/models/risk.py:16]: "reporter" might imply a journalist. maybe change to "as reported by a specific organization" ?

File: synapse/models/risk.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def getModelDefs(self):
                     'doc': 'A taxonomy of availability status values.',
                 }),
                 ('risk:tool:software', ('guid', {}), {
-                    'doc': 'A software tool used in threat activity.',
+                    'doc': 'A software tool used in threat activity, as described by a specific reporter.',
```
Comment (invisig0th) [inline synapse/models/risk.py:50]: and here too :)

File: synapse/models/risk.py [hunk 4]
```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
...
-                        'doc': 'The name of the organization who published the threat cluster.'}),
+                        'doc': 'The name of the organization reporting on the threat cluster.'}),
 
                     ('org', ('ou:org', {}), {
-                        'doc': 'The organization that the threat cluster is attributed to.'}),
+                        'doc': 'An authoritative organization for the threat cluster.'}),
 
                     ('org:loc', ('loc', {}), {
-                        'doc': 'The assessed location of the organization that the threat cluster is attributed to.'}),
+                        'doc': 'The reporter\'s assessed location of the threat cluster.'}),
```
Comment (invisig0th) [inline synapse/models/risk.py:125]: Gets a bit wordy, but may want to consider "The reporting orgnizations's" here and below?

File: synapse/models/risk.py [hunk 5]
```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
...
-                        'doc': 'The tag used to annotate nodes that are part of the tool subgraph.'}),
+                        'doc': 'The tag used to annotate nodes that are associated with the tool.'}),
 
                     ('desc', ('str', {}), {
-                        'doc': "A description of the tool's use in threat activity."}),
+                        'doc': 'A description of the tool.'}),
 
                     ('type', ('risk:tool:software:taxonomy', {}), {
-                        'doc': 'An analyst specified taxonomy of software tool types.'}),
+                        'doc': 'A user-specified type for the tool.'}),
```
Comment (invisig0th) [inline synapse/models/risk.py:162]: Another "not necessarily user defined" spot where we should probably use a standard taxonomy type description

File: synapse/models/risk.py [hunk 6]
```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
...
-                        'doc': 'The organization which reported the tool.'}),
+                        'doc': 'The organization reporting on the tool.'}),
 
                     ('reporter:name', ('ou:name', {}), {
-                        'doc': 'The name of the organization which reported the tool.'}),
+                        'doc': 'The name of the organization reporting on the tool.'}),
 
                     ('soft', ('it:prod:soft', {}), {
-                        'doc': 'The authoritative software family of the tool.'}),
+                        'doc': 'An authoritative software family for the tool.'}),
```
Comment (invisig0th) [inline synapse/models/risk.py:177]: This isn't "one of many" authoritative, so i think "the" is correct? Just like "the name of the organization" etc

File: synapse/models/risk.py [hunk 7]
```diff
@@ -397,10 +397,10 @@ def getModelDefs(self):
                 ('risk:alert:verdict:taxonomy', {}, {}),
                 ('risk:alert', {}, (
                     ('type', ('risk:alert:taxonomy', {}), {
-                        'doc': 'An alert type.'}),
+                        'doc': 'A user-specified alert type.'}),
```
Comment (invisig0th) [inline synapse/models/risk.py:400]: not user-specified

File: synapse/models/risk.py [hunk 8]
```diff
@@ -414,10 +414,10 @@ def getModelDefs(self):
 
                     ('verdict', ('risk:alert:verdict:taxonomy', {}), {
                         'ex': 'benign.false_positive',
-                        'doc': 'Analyst specified verdict taxonomy about why the alert is malicious or benign.'}),
+                        'doc': 'A user-specified verdict about why the alert is malicious or benign.'}),
```
Comment (invisig0th) [inline synapse/models/risk.py:417]: Should probably eliminate analyst/user specified here too. Could be machine specified as well.

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/models/risk.py [hunk 9]
```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
...
-                        'doc': 'The authoritative software family of the tool.'}),
+                        'doc': 'An authoritative software family for the tool.'}),
 
                     ('soft:name', ('it:prod:softname', {}), {
-                        'doc': 'The reported primary name of the tool.'}),
+                        'doc': 'The reporter\'s name for the tool.'}),
 
                     ('soft:names', ('array', {'type': 'it:prod:softname', 'uniq': True, 'sorted': True}), {
-                        'doc': 'An array of reported alterate names for the tool.'}),
+                        'doc': 'An array of alterate names for the tool, according to the reporter.'}),
```
Comment (reign-git) [inline synapse/models/risk.py:183]: alternate

File: synapse/models/risk.py [hunk 10]
```diff
@@ -101,86 +101,86 @@ def getModelDefs(self):
 
                     ('name', ('str', {'lower': True, 'onespace': True}), {
                         'ex': "apt1 (mandiant)",
-                        'doc': 'The name of the threat cluster.'}),
+                        'doc': 'A brief descriptive name for the threat cluster.'}),
 
                     ('type', ('risk:threat:type:taxonomy', {}), {
-                        'doc': 'The type of threat cluster.'}),
+                        'doc': 'A user-specified type for the threat.'}),
```
[the remark this answers is not in the document]
Comment (therealsilence) [reply synapse/models/risk.py:107]: Per discussion, "A type for the <foothing>, as a taxonomy entry"

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**